src/sidebar/util/test/watch-test.js

```diff
@@ -47,45 +47,42 @@ describe('sidebar/util/watch', () => {
       assert.notCalled(callback);
     });
 
-    it('compares watched values using strict equality', () => {
+    it('compares watched values using strict equality by default', () => {
       const callback = sinon.stub();
       const store = counterStore();
+      const getValue = () => [store.getState().a];
 
-      const newEmptyObject = () => ({});
-      watch(store.subscribe, newEmptyObject, callback);
+      watch(store.subscribe, getValue, callback);
 
-      store.dispatch({ type: 'INCREMENT_A' });
-      store.dispatch({ type: 'INCREMENT_A' });
+      store.dispatch({ type: 'INCREMENT_B' });
+      store.dispatch({ type: 'INCREMENT_B' });
```

Comment:
Just curious why this change?

Reply:
This is to create a state where the result of `getValue` changes each time if a strict-equality check is used (which we are), but not if a deep equality check were used (which we are not). Here the store states looks like:

1. `{ a: 0, b: 0 }`
2. `{  a: 0, b: 1 }`
3.  `{ a: 0, b: 2 }`

And the sequence of `getValue` results is `[0]`, `[0]`, `[0]`.